src/containers/forgot-password-reset/index.tsx

```diff
@@ -6,15 +6,29 @@ import authClient from '../../auth/authClient';
 
 const { Title } = Typography;
 
+interface NewPasswords {
+  readonly password: string;
+  readonly confirmPassword: string;
+}
+
 const ForgotPasswordReset: React.FC = () => {
   const { key } = useParams();
 
-  // gotta change this
-  const onFinish = (values: any) => {
-    authClient.forgotPasswordReset({
-      secretKey: key,
-      newPassword: values.password,
-    });
+  const onFinish = (values: NewPasswords) => {
+    if (values.password.length < 8) {
+      alert('New password is too weak. Must be at least 8 characters long.');
+    } else if (values.password !== values.confirmPassword) {
+      alert("Passwords don't match");
+    } else {
+      authClient
+        .forgotPasswordReset({ secretKey: key, newPassword: values.password })
+        .then(() => {
+          alert('Successfully reset password!');
```

Comment:
does eslint complain about `alert`? I think it might be best to just navigate to a barebones "You successfully reset your password!" page, or swap out the rendered JSX to say something similar.